docs/cli_hacking_guide.adoc

```diff
@@ -199,7 +221,26 @@ $ hack/update-generated-docs.sh
 
 === Handling Errors
 
-TODO
+When delcaring the `Run:` field in the cobra comand, make sure to call the `Complete`, `Validate`, `Run` methods
+within the `k8s.io/kubernetes/pkg/kubectl/cmd/util#CheckErr` helper, which will take care of exiting with the correct
+exit code in the event of an error:
+
+====
+[source,go,options="nowrap"]
+----
+cmd := &cobra.Command{
+  Use:     "foo [flags]",
+  Short:   "short command description",
+  Long:    descLong,
+  Example: fmt.Sprintf(fooExample, fullName),
+  Run: func(cmd *cobra.Command, args []string) {
+    kcmdutil.CheckErr(o.Complete(f, cmd, args))
+    kcmdutil.CheckErr(o.Validate())
+    kcmdutil.CheckErr(o.Run())
+  },
+}
+----
+====
 
 === Tests
```

Comment:
Remove the section about tests, at least for now. When we'll write it we'll add it :-)